corehq/apps/case_importer/views.py

```diff
@@ -135,30 +182,24 @@ def _process_file_and_get_upload(uploaded_file_handle, request, domain, max_colu
     request.session[EXCEL_SESSION_ID] = case_upload.upload_id
 
     case_upload.check_file()
-    invalid_column_names = set()
-    with case_upload.get_spreadsheet() as spreadsheet:
-        columns = spreadsheet.get_header_columns()
-        validate_column_names(columns, invalid_column_names)
-        row_count = spreadsheet.max_row
 
-    if invalid_column_names:
-        error_message = format_html(
-            _("Column names must be <a target='_blank' href='https://www.w3schools.com/xml/xml_elements.asp'>"
-              "valid XML elements</a> and cannot start with a number or contain spaces or most special characters."
-              " Please update the following: {}.").format(
-                ', '.join(invalid_column_names)))
-        raise ImporterRawError(error_message)
-
-    if row_count == 0:
-        raise ImporterError(_('Your spreadsheet is empty. Please try again with a different spreadsheet.'))
+    worksheet_titles = _get_workbook_sheet_names(case_upload)
+    case_types_from_apps = sorted(get_case_types_from_apps(domain))
 
-    if max_columns is not None and len(columns) > max_columns:
-        raise ImporterError(_(
-            'Your spreadsheet has too many columns. '
-            'A maximum of %(max_columns)s is supported.'
-        ) % {'max_columns': MAX_CASE_IMPORTER_COLUMNS})
+    # It is a bulk import if every sheet name is a case type in the project space.
+    # This does introduce the limitation that new cases for new case types cannot be bulk imported
+    is_bulk_import = len(set(worksheet_titles) - set(case_types_from_apps)) == 0
```

Comment:
This is totally a personal preference and just the way my brain works, so feel free to ignore, but curious if your brain also finds this more readable:
```suggestion
    is_bulk_import = all(t in case_types_from_apps for t in worksheet_titles)
```
(It could be slower for large sets (I didn't bother checking) but not a real hit on sets this small.)

Reply:
I strangely find the other way easier to read :smile:. This could just be me however, and I'm curious if others find this less readable as well. If this is the case, then it might be worth changing for overall readability. 

What are your thoughts on this @SmittieC?